.devcontainer/docker-compose.yml

```
@@ -0,0 +1,66 @@
+version: '3'
```

When I follow your blog post, mine generates with a different version. Does it matter?

```suggestion
version: '3.8'
```

---

Old habits die hard for me, for maintainability of the docker-compose file it's best to just keep the full version used, thanks!